src/about/index.md

```diff
@@ -2,7 +2,8 @@
 
 Welcome to the Void Handbook! Please be sure to read the "[About This
 Handbook](./about-this-handbook.md)" section to learn how to use this
-documentation effectively.
+documentation effectively. A local copy of this handbook in several formats can
```

Comment:
Would prefer: "handbook, in several formats, can"

Reply:
Fixed!